pkg/util/strategicpatch/patch.go

```diff
@@ -145,7 +174,11 @@ func mergeMap(original, patch map[string]interface{}, t reflect.Type) (map[strin
 // Merge two slices together. Note: This may modify both the original slice and
 // the patch because getting a deep copy of a slice in golang is highly
 // non-trivial.
-func mergeSlice(original, patch []interface{}, elemType reflect.Type, mergeKey string) ([]interface{}, error) {
+func mergeSlice(original, patch []interface{}, elemType reflect.Type, mergeKey string, dw *diffWriter) ([]interface{}, error) {
+	dw.DepthInc()
+	defer dw.DepthDec()
+	fmt.Printf("XXX")
```

Comment:
Did you mean to leave this here? If so, you should be writing to diffWriter's buffer, not to stdout directly.